atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -43,6 +44,9 @@ def __init__(self, tasker, workflow, image_id, ignore_autogenerated_gpg_keys=Tru
         self.ignore_autogenerated_gpg_keys = ignore_autogenerated_gpg_keys
 
     def run(self):
+        if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+            self.log.debug("image has scratch base, do not perform RPM check")
+            return
```

Comment:
We'd still want to run this though wouldn't we? I'm not sure why we would skip it.